chromium_src/components/sync/engine_impl/commit.cc

```diff
@@ -30,14 +30,15 @@ SyncerError PostBraveCommit(sync_pb::ClientToServerMessage* message,
 #include "brave/components/brave_sync/jslib_const.h"
 #include "brave/components/brave_sync/jslib_messages.h"
 #include "brave/components/brave_sync/jslib_messages_fwd.h"
+#include "brave/components/brave_sync/tools.h"
 #include "components/sync/base/time.h"
 #include "components/sync/base/unique_position.h"
 
 namespace syncer {
 namespace {
 using brave_sync::jslib::MetaInfo;
 using brave_sync::jslib::SyncRecord;
-const char kBookmarkBarTag[] = "bookmark_bar";
+const char kOtherBookmarksTag[] = "other_bookmarks";
```

Comment:
this constant is taken from `src/components/sync_bookmarks/bookmark_model_merger.cc` and marks other bookmarks  node entity

Reply:
comment added